Releasenotes.md

```diff
@@ -1,6 +1,68 @@
 # Release notes BRK-Bevragen
 
-## Versie 1.1.1:
+## **Versie 1.2.0:**
+
+### openapi.yaml
+
+- Algemeen
+  - Diverse verbeteringen in de documentatie (descriptions) en examples.
+  - Er is hergebruik gemaakt van Haal-Centraal-common versie 1.2.0 (was 1.0.0)
+  - De default-waarde voor de Accept-Crs headerparameter is verwijderd.
+  - Uit alle componentnamen zijn de underscores verwijderd.
+
+
+- KadastraalOnroerendeZaak :
+  - De default-waarde voor de Accept-Crs headerparameter is verwijderd bij /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}
+  - PrivaatrechtelijkeBeperkingen, hypotheken en beslagen zijn toegevoegd aan de _embedded en _links. De bijbehorende identificaties zijn opgenomen in de KadastraalOnroerendeZaak.
+  - Bij LocatieKadastraalObject wordt nu het adres meegeleverd waar dat voorheen alleen een verwijzing met een nummeraanduidingIdentificatie was.
+
+
+- Zakelijkgerechtigden
+  - Accept-Crs header parameter en content-CRS header zijn toegevoegd bij /kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}/zakelijkgerechtigden/{zakelijkgerechtigdeidentificatie} en bij kadastraalonroerendezaken/{kadastraalonroerendezaakidentificatie}/zakelijkgerechtigden
```

Comment:
_"header parameter"_ of _"headerparameter"_ (consistentie) of toch gewoon _"header"_?